docs/collect-intro.rst

```diff
@@ -3,24 +3,35 @@ ODK Collect
 
 .. _collect-introduction:
 
-:dfn:`ODK Collect` is an open source Android app that replaces paper forms used in survey-based data gathering. It supports a wide range of question and answer types, and is designed to work well without network connectivity.
+:dfn:`ODK Collect` is an open source Android app for managing and filling out forms. It supports a wide range of question and answer types, and is designed to work well without network connectivity.
 
-ODK Collect renders forms into a sequence of input prompts that apply form logic, entry constraints, and repeating sub-structures. Users work through the prompts and can save the submission at any point. Finalized submissions can be sent to (and new forms downloaded from) a server. 
+ODK Collect is used by connecting to a server used to download blank forms and submit filled ones. Collect displays forms as input prompts that can include logic, constraints, and repeating sub-structures. Users work through the prompts and can save their data at any point.
 
 Collect supports location, audio, images, video, barcodes, signatures, multiple-choice, free text, and numeric answers. It can even accept answers from other apps on your device. :doc:`See a complete list of supported question types here.  <form-question-types>`
 
-.. image:: /img/form-question-types/string-input.*
-  :alt: String input form widget, displayed in ODK Collect on an Android phone. The label is "What is your name?"
+.. image:: /img/collect-intro/afp-knowledge.*
+  :alt: Knowledge assessment for accute flaccid paralysis.
   :class: device-screen-vertical side-by-side
-.. image:: /img/form-question-types/single-select.*
-  :alt: Single-select (radio button) form widget, displayed in ODK Collect on an Android phone. The question label is "What is your favorite fruit?" After the question is a list of fruits.
+.. image:: /img/collect-intro/geoshape.*
+  :alt: Capturing a polygon on a map.
   :class: device-screen-vertical side-by-side
 
+.. _collect-supported-devices:
+
+Supported devices
+-------------------
+
+You can use Collect with a wide range of Android devices and the only firm requirement is that they must use Android 5.1 or above. That said, we generally recommend using Android 10 or higher for the best :doc:`security <collect-security>` and performance.
+
+Collect's performance depends on the forms that it is used with. If your forms include mostly simple questions without much logic, they will work well on any device.
```

Comment:
I'd drop this whole thing. The first sentence is addressed in the next paragraph, which is quite good, and the second sentence is oddly vague and thus scary.